server/start.go

```diff
@@ -141,6 +143,7 @@ which accepts a path for the resulting pprof file.
 	cmd.Flags().Uint64(server.FlagPruningInterval, 0, "Height interval at which pruned heights are removed from disk (ignored if pruning is not 'custom')")
 	cmd.Flags().Uint(server.FlagInvCheckPeriod, 0, "Assert registered invariants every N blocks")
 	cmd.Flags().Uint64(server.FlagMinRetainBlocks, 0, "Minimum block height offset during ABCI commit to prune Tendermint blocks")
+	cmd.Flags().String(srvflags.AppDBBackend, "", "The type of database for application and snapshots databases")
```

Comment:
what's the default here?

Reply:
there's a fallback chain:
- the old compile flag `github.com/cosmos/cosmos-sdk/types.DBBackend`
- the `db_backend` in `config.toml`